src/test/fixtures/sample-config-rinkeby.json

```diff
@@ -422,7 +422,8 @@
       "attachments": {
         "allow": true,
         "accept": ".pdf"
-      }
+      },
+      "extension": "test"
```

Comment:
Might want to change the extension to "tt" as the config file will be given to the front-end desk.

Reply:
ok